test/functional/tests/incremental_load/test_incremental_load.py

```diff
@@ -0,0 +1,646 @@
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_cache_with_inactive_core():
+    """
+        title: Load cache with unavailable core devices.
+        description: Check if it is possible to load cache with unavailable core devices.
+        pass_criteria:
+          - No kernel error
+          - It is possible to perform cache load operation with unavailable devices.
+          - Warning message about not available core device should appear.
+    """
+    with TestRun.step("Prepare devices."):
+        devices = prepare_devices([("cache", 1), ("core", 1)])
+        cache_dev = devices["cache"].partitions[0]
+        core_dev = devices["core"].partitions[0]
+        plug_device = devices["core"]
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_dev, force=True)
+        cache.add_core(core_dev)
+    with TestRun.step("Create init config file using current CAS configuration."):
+        init_config.create_init_config_from_running_configuration()
+    with TestRun.step("Stop cache."):
+        cache.stop()
+    with TestRun.step("Unplug core device."):
+        plug_device.unplug()
+    with TestRun.step("Load cache."):
+        output = TestRun.executor.run(cli.load_cmd(cache_dev.system_path))
+        cli_messages.check_msg(output, cli_messages.load_inactive_core_missing)
+    with TestRun.step("Plug missing device and stop cache."):
+        plug_device.plug()
+        cache.stop()
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_preserve_data_for_inactive_device():
+    """
+        title: Validate preserving data for inactive CAS devices.
+        description: Validate that cached data for inactive CAS devices is preserved.
+        pass_criteria:
+          - No kernel error
+          - File md5 checksums match in every iteration.
+          - Cache read hits increase after reads (md5 checksum) from CAS device with attached core.
+    """
+    mount_dir = "/mnt/test"
+    with TestRun.step("Prepare devices."):
+        devices = prepare_devices([("cache", 1), ("core", 1)])
+        cache_dev = devices["cache"].partitions[0]
+        core_dev = devices["core"].partitions[0]
+        plug_device = devices["core"]
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_dev, cache_mode=CacheMode.WB, force=True)
+        core = cache.add_core(core_dev)
+    with TestRun.step("Create init config file using current CAS configuration."):
+        init_config.create_init_config_from_running_configuration()
+    with TestRun.step("Create filesystem on CAS device and mount it."):
+        core.create_filesystem(Filesystem.ext3)
+        core.mount(mount_dir)
+    with TestRun.step("Create a test file with random writes on mount point and count it's md5."):
```

Comment:
I think you should set `seq_cutoff_policy` to `never` if you're using `dd` - most of the workload doesn't get cached

Reply:
Done